conductr_cli/endpoint.py

```diff
@@ -0,0 +1,115 @@
+from enum import Enum
+from pyhocon import ConfigTree, ConfigList
+
+
+class Endpoint:
+    def __init__(self, endpoint_dict):
+        self.name = endpoint_dict['name']
+        self.component = endpoint_dict['component']
+
+        if 'bind-protocol' in endpoint_dict:
+            self.bind_protocol = endpoint_dict['bind-protocol']
+        else:
+            self.bind_protocol = 'tcp'
+
+        if 'bind-port' in endpoint_dict:
+            self.bind_port = endpoint_dict['bind-port']
+        else:
+            self.bind_port = 0
+
+        if 'service-name' in endpoint_dict:
+            self.service_name = endpoint_dict['service-name']
+        else:
+            self.service_name = None
+
+        self.acls = []
+        if 'acls' in endpoint_dict:
+            for acl in endpoint_dict['acls']:
+                if acl.startswith('http'):
+                    self.bind_protocol = 'http'
+                self.acls.append(RequestAcl(acl))
+
+    def hocon(self):
+        endpoint_tree = ConfigTree()
+        endpoint_tree.put('{}.bind-protocol'.format(self.name), self.bind_protocol)
+        endpoint_tree.put('{}.bind-port'.format(self.name), self.bind_port)
+        if self.service_name:
+            endpoint_tree.put('{}.service-name'.format(self.name), self.service_name)
+        for acl in self.acls:
+            acl_list = ConfigList()
+            acl_list.append(acl.hocon())
+            endpoint_tree.put('{}.acls'.format(self.name), acl_list)
+        return endpoint_tree
+
+
+class RequestAcl:
+    request_mappings = []
```

Comment:
`request_mappings` need to be declared inside `__init__` as:

```
self.request_mappings = []
```

I'm assuming you'd like to declare this as instance field, not static field.

Sorry for missing this earlier.